### PR TITLE
Feature/api headers

### PR DESCRIPTION
**Pull request by dmytrokhl**
## Add data to headers
https://app.clubhouse.io/vgs/story/53185/send-additional-data-in-ios-sdk-headers

## Description of changes in release / Impact of release:
Update API headers data.

### CODE REVIEW

File: framework/Sources/VGSFramework/Core/APIClient.swift [hunk 1]
```diff
@@ -11,23 +11,38 @@ import Alamofire
...
-        var headers = [
-            "Content-Type": "application/json",
-            "vgs-client": "source=iosSDK&medium=vgs-collect&content=1.0"
+        let version = ProcessInfo.processInfo.operatingSystemVersion
+        let versionString = "\(version.majorVersion).\(version.minorVersion).\(version.patchVersion)"
+        let vgsCollectVersion: String = {
+            guard
+                let vgsInfo = Bundle(for: APIClient.self).infoDictionary,
+                let build = vgsInfo["CFBundleShortVersionString"]
+            else { return "Unknown" }
```
Comment (miraving) [inline framework/Sources/VGSFramework/Core/APIClient.swift]: Oy wey!
What is the formatting style name? 😜
I believe `swiftlint` not crashing

File: framework/Sources/VGSFramework/Core/APIClient.swift [hunk 2]
```diff
@@ -11,23 +11,38 @@ import Alamofire
...
+        return [
+            "vgs-client": "source=iosSDK&medium=vgs-collect&content=\(vgsCollectVersion)&osVersion=\(versionString)"
         ]
-        
+    }()
+    
+    func sendRequest(path: String, method: HTTPMethod = .post, value: BodyData, completion block: @escaping (_ data: JsonData?, _ error: Error?) -> Void) {
+        // Add Headers
+        var headers = APIClient.defaultHttpHeaders
+        headers["Content-Type"] = "application/json"
```
Comment (miraving) [inline framework/Sources/VGSFramework/Core/APIClient.swift]: The Alamofire is setting the `application/json` from the box.

File: framework/Sources/VGSFramework/Core/Enums.swift [hunk 3]
```diff
@@ -22,20 +22,20 @@ public enum Environment: String {
...
 /// - dateExpiration: Expiration date with formatting and validation
-/// - cvv: CVV code for card.
+/// - cvc: CVC code for card.
 /// - cardHolderName: Card holder name
 public enum FieldType: Int, CaseIterable {
     case none
     case cardNumber
     case expDate
-    case cvv
+    case cvc
```
Comment (miraving) [inline framework/Sources/VGSFramework/Core/Enums.swift]: Why cvc? I know CVV ☹️

Comment (dmytrokhl) [reply framework/Sources/VGSFramework/Core/Enums.swift]: In US mostly used CVC, same in our JS library, so we make decision to change cvv -> cvc.

**Pull request merged**